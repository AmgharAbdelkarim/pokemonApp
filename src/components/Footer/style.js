import styled from "styled-components";

export const FooterWrapper = styled.div`
  padding: 20px;
  background-color: #ffffff;
  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.25);
  text-align: end;
`;
