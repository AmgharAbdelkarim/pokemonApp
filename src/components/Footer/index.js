import React from "react";
import { FooterWrapper } from "components/Footer/style";
const Footer = () => {
  return (
    <FooterWrapper id="footer">
      <span>All rights reserved Â© 2020</span>
    </FooterWrapper>
  );
};
export default Footer;
