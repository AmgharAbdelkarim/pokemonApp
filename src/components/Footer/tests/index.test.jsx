import { cleanup, render } from "@testing-library/react";

import Footer from "components/Footer";

afterEach(cleanup);

describe("Footer", () => {
  it("should Footer render correctly", () => {
    const { asFragment, getByText } = render(Footer());
    expect(getByText("All rights reserved Â© 2020")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
