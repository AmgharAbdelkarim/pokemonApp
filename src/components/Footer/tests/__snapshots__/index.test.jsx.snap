// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should Footer render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ jckeWk"
    id="footer"
  >
    <span>
      All rights reserved Â© 2020
    </span>
  </div>
</DocumentFragment>
`;
